homeassistant/components/http/__init__.py

```diff
@@ -296,6 +298,7 @@ def __init__(
 
         # This order matters
         setup_real_ip(app, use_x_forwarded_for, trusted_proxies)
+        setup_request_context(app, current_request)
```

Comment:
Shouldn't this be the first middleware ?

Reply:
Actually, I want a second step after this one to get rid of the one above that. In this case, the order between these doesn't matter.

Reply:
Swapped.